.editorconfig

```diff
@@ -0,0 +1,12 @@
+# top-most EditorConfig file
+root = true
```

Comment:
Should this file be committed?

Reply:
I think it's nice to have, most editors should be ale to detect it and use its values to adjust parameters of number of spaces for indentation, etc. I can remove it if you'd like but I find it pretty useful (we have it on some other repos like nav-ext).